### ISSUE TITLE
Feature request: Remember last save location

### ISSUE THREAD

**Issue by Nadahar** [qTox/qTox#4971]
I use qTox quite a lot, and I'm generally very pleased with it. It fills the role for me that Skype once did (before it became too bloated/Microsoft'ed) while being encrypted on top :+1: 

There are some small things here and there, but the only one that really bugs me is the save location. I use qTox while developing, exchanging a lot of log files, images, configuration files, documentation and whatnot. The idea of dumping all of this in one folder doesn't make much sense in my case, so I constantly need to browse my way to where I want to save a file. I often transfer several related files when I work with something specific, and having the option to reuse the last save location would help a lot.

I think that options like what Firefox offers would be perfect:

![savelocation](https://user-images.githubusercontent.com/12900583/36634585-020c0f9c-19a7-11e8-9809-00be212176ee.png)

There's also a small annoyance with the browsing itself, but I suspect that's due to Qt itself and not qTox. This is that whenever you click or via the keyboard highlight any file in the save dialog, the name of the file to save is changed to that. I usually "type to search" my way to the folders when I navigate to find the correct location, and while typing there's no way to avoid if a file might be selected briefly - resulting in that the name of the file to save is lost.

Combining the two issues described above can make saving files a hassle, I generally have to copy the file name to the clipboard before browsing and then paste it again. If I forget to copy it, I have to abort the transfer and request the other party to resend the file.

This seems to have been discussed in  #2062, without leading to a solution.

OS: Windows
qTox version: v1.13.0
Commit hash: 531defd0aa66af1b128f7293bc08718ce2cc064f 
toxcore: 0.1.10
Qt: 5.9.3

**Comment by sudden6** [comment]
I agree, this is annoying me to, probably #2062 should be reopened and updated.

**Comment by tox-user** [comment]
> I think that options like what Firefox offers would be perfect

Which option should be the default?

**Comment by sudden6** [comment]
I think default should be to ask
